duckduckgo/iOS
# Update project to run with XCode 9

Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/449517770398660

**Description**:
Updates the app to run on XCode9. This includes:
• updating project settings
• updating to Swift 4
• ensuring unit tests run both locally and on travis

**Steps to test this PR**:
Check that the app runs locally. Check that unit tests run via the XCode, via the command line and on travis.

Note: the UI tests were updated to ensure that they build and run but not to necessarily pass as this needs to happen after the XCode 9 UI changes are in place.

###### Reviewer Checklist:


###### PR DRI Checklist:
